cis/packer.json

```diff
@@ -2,9 +2,9 @@
   "variables": {
     "region": "us-east-1",
     "instance_type": "t3.medium",
-    "associate_public_ip": "true",
```

Comment:
Why are you defaulting to no public IP for the builder? That can make it a lot more annoying to build from a laptop.